packages/flutter/lib/src/widgets/app.dart

```diff
@@ -972,6 +975,18 @@ class WidgetsApp extends StatefulWidget {
   /// {@endtemplate}
   final String? restorationScopeId;
 
+  /// {@template flutter.widgets.widgetsApp.scrollBehavior}
+  /// The default [ScrollBehavior] for the application.
```

Comment:
Would probably be good to have a little more context in this doc explaining, e.g., when one would want to provide their own behavior?

Reply:
For sure! Thank you! This is still a rough draft, a lot of docs and tests will come if we decide to go in this direction. :)